src/config.py

```diff
@@ -80,7 +104,7 @@ class ProtocolFeeConfig:
     def from_network(network: Network) -> ProtocolFeeConfig:
         """Initialize protocol fee config for a given network."""
         match network:
-            case Network.MAINNET:
+            case Network.MAINNET | Network.GNOSIS | Network.ARBITRUM_ONE:
```

Comment:
@fhenneke This probably needs updating?

Reply:
I added appropriate addresses for each chain.